### ISSUE TITLE
Backup to multiple locations

### ISSUE THREAD

**Issue by anaxonda** [stevesoltys/backup#340]
Hello,

Is it possible to set seedvault to back up to more than one location? Right now I only see it is possible to choose one location.

Thanks

**Comment by grote** [comment]
This would be possible to do, but every location makes it more likely to run into AOSP backup API timeouts. So I'd be hesitant to even implement this feature as we are already stretching those timeouts.

**Comment by nfriedly** [comment]
I want to backup to my microsd and to nextcloud - so for that scenario, perhaps it could backup to the microsd first, and then after that's complete, it could just copy that data to nextcloud without worrying about time limits?

**Comment by grote** [comment]
Couldn't you use another app to regularly backup/copy/sync the primary folder to the other locations?

**Comment by nfriedly** [comment]
Sure, probably. But it seems like a reasonable thing to want to have built-in.

**Comment by stevleibelt** [comment]
@nfriedly It would be cool if you could spend some time finding a tool out there that is doing your copy-paste thing.

If there is a tool out there doing what it is needed, the maintainers here don't have to reinvent the wheel here and just had to add a how to section in the README - Or you could add this how to section and create a pull request :-).

If you don't find any time or the importance of this feature gained some down steps, it would be cool if you could think about closing this ticket since @grote already explained that this feature would not be easy to implement and is possible low on the priority list.

**Comment by nettnikl** [comment]
I disagree, the issue is not solved i.e. the feature is not implemented by 'stepping down its priority'. Let's not do that - there is no SLA to be met by closing ticket prematurely.

If it is not currently urgently needed, it might be ignored as a feature request. And especially since it seems hard to implement, lets rather move this to the github discussions, so trouble-with and ways-to implement it can be discussed.

**Comment by nfriedly** [comment]
Yeah, it's their repo and they can do what they like, but I'm in favor of leaving it open as a place to collect ideas and feedback on the issue.

It looks like at least one other issue (https://github.com/seedvault-app/seedvault/issues/399) was closed as a duplicate of this one.

I've stopped using SeedVault for the time being for a couple of reasons. 1) it was filling up my nextcloud ssd and 2) I wasn't sure how to inspect the backups and verify their contents or pull out individual files. I personally don't need or even want encrypted backups; I'd prefer something that was easier to inspect and pull specific files from. So SeedVault probably isn't the right solution for me. With that in mind, y'all are welcome to do whatever you'd like with this issue.

**Comment by stevleibelt** [comment]
@nfriedly to what software or backup strategy have you switched to?

I am using syncthing for all media related files. But for app's with there settings, I haven't found my silver bullet while using Linageos 20. This to be written, Seedvault is currently not capable of transferring all apps and settings from one poco x3 to a new one (with a working camera). Because of this fact, I've started searching for a non-google alternative. Also noted, it looks like Android 13 made it harder to have a non-google-backup-system that is working without issues :-/.

**Comment by nfriedly** [comment]
* I turned Google back on for most things. 
* I use [SMS Backup Restore](https://play.google.com/store/apps/details?id=com.riteshsahu.SMSBackupRestore) to backup my text messages and call logs (because Google seems to skip those for some reason). 
* I have Nextcloud automatically uploading any new photos or videos (so, they go to both Google Photos and NC).
* I made a one-off backup of my contacts to nextcloud.
